backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/DebugUtils.kt

```diff
@@ -63,6 +63,8 @@ fun KonanConfig.debugInfoVersion():Int = configuration[KonanConfigKeys.DEBUG_INF
 internal class DebugInfo internal constructor(override val context: Context):ContextUtils {
     val files = mutableMapOf<String, DIFileRef>()
     val subprograms = mutableMapOf<LLVMValueRef, DISubprogramRef>()
+    /* Some functions inlined out and haven't got LLVM value. */
```

Comment:
"Inlined out" :). Maybe something like
"Some functions are inlined on all callsites and body is eliminated by DCE, so there's no LLVM value"?